app/test_main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from unittest import mock
+
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.has_internet_connection")
+def test_has_been_has_internet_connection_called(
```

Comment:
Let's add better test name. Example `test_internet_connection_called`

Rename second test the same way